template/config/webpack.prod.config.js

```diff
@@ -177,9 +134,12 @@ module.exports = {
     // Creates HTML page for us at build time
     new HtmlWebpackPlugin(),
 
-    // An ES6+ aware minifier, results in smaller output compared to UglifyJS given that 
+    // An ES6+ aware minifier, results in smaller output compared to UglifyJS given that
     // Chromium in electron supports the majority of ES6 features out of the box.
-    new MinifyPlugin()
+    new MinifyPlugin(),
```

Comment:
why did you add the comma here? Isn't it the last element in the array?

Reply:
Ah, got it, because of the commented out BundleAnalyzerPlugin. nevermind.

Reply:
So that you can simply uncomment the line below it if you want to run the analyser over the build.

@rueckstiess I can remove the dangling comma if you prefer?